### DISCUSSION TITLE
Os Keyring couldnt be identified for storing the encryption!

### DISCUSSION THREAD

**Discussion by LazyButLearning** [dat-linux/community#37] [FAQ] [answered]
When ever i open VSCodium i get this error "An OS key-ring couldn't  be identified for storing the encryption related data in your current desktop environment."

![screen](https://github.com/dat-linux/community/assets/140408915/29aea3ab-0f27-4b08-9f3a-1fa342fc417e)

First step was to google it in various websites and there seem to have many answers related to vs code not vscodium, yes both are similar but yet different.

so i installed vs code and checked as all  the answers were related to that. it only worked when i installed gnome key-ring and set key-ring to gnome.

As Dat Linux is in LXQT desktop is there any way for os keyring to work in LXQT desktop for either vs-code or vscodium ??

**Comment by dat-linux** [comment]
Thanks for reporting. I do not see this error on a fresh install. And gnome-keyring is definitely pre-installed in the distro.

**Comment by dat-linux** [comment, marked as answer]
Cannot replicate, closing.
